src/azul/service/manifest_service.py

```diff
@@ -326,6 +334,7 @@ def get_manifest(self,
                      catalog: CatalogName,
                      filters: Filters,
                      partition: ManifestPartition,
+                     auth_token: Optional[str],
```

Comment:
So this for example should read
```suggestion
                     auth: Optional[Authentication],
```